lib/pbench/server/database/models/users.py

```diff
@@ -91,24 +98,38 @@ def evaluate_role(self, key: str, value: str) -> Optional[str]:
         except KeyError:
             return None
 
-    @validates("password")
-    def evaluate_password(self, key: str, value: str) -> str:
-        return generate_password_hash(value)
-
     # validate the email field
     @validates("email")
     def evaluate_email(self, key: str, value: str) -> str:
         valid = validate_email(value)
         return valid.email
 
+    def updateDict(self, orig_dict: JSON, new_dict: JSON) -> JSON:
+        """Update the Original json dict with a new json dict
+        Args:
+            orig_dict: Original JSON object to update
+            new_dict: New JSON object to parse
+        Returns:
+            Update JSON object
+        """
+        for key, val in new_dict.items():
+            if isinstance(val, dict):
+                tmp = update(orig_dict.get(key, {}), val)
+                orig_dict[key] = tmp
```

Comment:
Should that be `self.update()`?  Or, more likely, 
```
            if isinstance(val, dict):
                if key not in orig_dict:
                    orig_dict[key] = {}
                orig_dict[key].update(val)
```
Note that this makes a shallow copy of each of the values in `val`...hopefully, that kind of sharing is OK....

And, as Dave pointed out, we run into problems if `orig_dict[key]` is not a `dict`....

Reply:
No longer relevant, I removed this block of code.